sentry-core/src/metrics.rs

```diff
@@ -1180,4 +1185,11 @@ mod tests {
         assert!(metrics.contains(&format!("my.metric:1|c|T{ts}")));
         assert!(metrics.contains(&format!("my.dist:2|d|T{ts}")));
     }
+
+    #[test]
+    fn test_regression_parse_statsd() {
+        let payload = "docker.net.bytes_rcvd:27763.20237096717:27763.20237096717:27763.20237096717:27763.20237096717:1|g|#container_id:97df61f5c55b58ec9c04da3e03edc8a875ec90eb405eb5645ad9a86d0a7cd3ee,container_name:app_sidekiq_1,docker_image:ghcr.io/mastodon/mastodon:nightly.2024-02-02-security,docker_network:app_internal_network,git.commit.sha:1726085db5cd73dd30953da858f9887bcc90b958,git.repository_url:https://github.com/mastodon/mastodon,host:aaaa,image_name:ghcr.io/mastodon/mastodon,image_tag:nightly.2024-02-02-security,runtime:docker,short_image:mastodon,source_type_name:System,environment:production|T1707081890";
+        let metric = Metric::parse_statsd(payload).unwrap();
+        assert_eq!(metric.name, "docker.net.bytes_rcvd");
```

Comment:
it we really want to preserve "roundtrips" (which this does not do fully), we should check the fully serialized metric.
also no need to put *all* the tags into this test :-)

Reply:
The question for me is also whether we should be more general in parsing: do we _want_ to discard everything but the `last` value, or would it make sense to parse and preserve them?